app/controllers/links_controller.rb

```diff
@@ -38,6 +40,14 @@ def documentation_url(section = "", options = {})
     root_url + (section || '')
   end
 
+  # For new documentation at docs.theforeman.org
+  def new_documentation_url(section, chapter)
+    version = SETTINGS[:version].short
+    chapter_hash = "##{chapter}" if chapter
+
+    "https://docs.theforeman.org/#{version}/#{section}/index-foreman-el.html#{chapter_hash}"
```

Comment:
It appears that not all of them use `index-foreman-el.html`, some are `index-foreman-deb.html` or `index-katello.html`

Reply:
Yeah its a bit messy as we build multiple versions per guide and not all of them are relevant. To see what is meaningful, visit the index page to see the links. I think this method needs a third argument which is the "suffix" which can be:

* foreman-el
* satellite (not relevant for upstream)
* katello
* foreman-deb
* orcharhino

Reply:
All